src/mpid/ch4/src/ch4r_proc.h

```diff
@@ -87,70 +87,67 @@ MPL_STATIC_INLINE_PREFIX int MPIDIU_comm_rank_to_pid(MPIR_Comm * comm, int rank,
 #define FCNAME MPL_QUOTE(FUNCNAME)
 MPL_STATIC_INLINE_PREFIX MPIDI_av_entry_t *MPIDIU_comm_rank_to_av(MPIR_Comm * comm, int rank)
 {
-    MPIDI_av_entry_t * ret;
+    MPIDI_av_entry_t * ret = NULL;
```

Comment:
I intentionally avoided this initialization which did not exist in the original code path. I have a feeling that this shouldn't be a huge performance penalty though. For the code readability I prefer your approach. Maybe @yfguo can comment.